src/scripts/fixtures_script.sql

```diff
@@ -0,0 +1,61 @@
+-- FUNCTION : product_fixture
+
+CREATE PROCEDURE build_product(
+    		    				p_name varchar(50) default null,
+    		    				p_description varchar(50) default null,
+    							p_entity_status varchar(50) default null,
+    							p_create_by varchar(50) default null,
+    							p_create_date timestamp without time zone default null,
+    							p_update_by varchar(50) default null,
+    							p_update_date timestamp without time zone default null
+                              )
+	  LANGUAGE 'plpgsql'
+	  		AS $$
+	  		
+		 BEGIN
+		 	 
+				IF p_name IS NULL THEN
```

Comment:
Could you use default values instead of using all these if conditions?

Reply:
done.